examples/dummy_honest/examples/dummy_honest.rs

```diff
@@ -0,0 +1,217 @@
+use aleph_bft::{run_session, Recipient};
```

Comment:
Regarding the path to the current file: why nested examples again :P? I presume this is on purpose, but why not just put this into `main.rs`?

Reply:
as above